openbas-front/src/admin/components/atomic_testings/atomic_testing/Index.tsx

```diff
@@ -106,12 +107,24 @@ const Index = () => {
                 label={t('Execution details')}
                 className={classes.item}
               />
+              {
+                injectResultOverviewOutput.inject_type !== 'openbas_email' && injectResultOverviewOutput.inject_type !== 'openbas_ovh_sms' && (
```

Comment:
You missed the Mastodon Inject
![image](https://github.com/user-attachments/assets/0350a940-abe2-40fe-a230-2037e537b7bb)